group/src/main/java/com/android/group/model/firebase/Group.java

```diff
@@ -3,15 +3,120 @@
 import java.util.List;
 
 public final class Group {
-    public String groupName;
-    public String category;
-    public String iD;
-    public List<Object> userList;
-    public String description;
-    public int users;
-    public double lat;
-    public double lng;
-
-    public Group() {
+    private final String groupName;
+    private final String category;
+    private final String iD;
+    private final List<Object> userList;
+    private final String description;
+    private final int users;
+    private final double lat;
+    private final double lng;
+
```

Comment:
in order for firebase to use this model it needs a default constructor

Reply:
ill added it